deploy/chaos/loadtest/Makefile

```diff
@@ -0,0 +1,8 @@
+URL := "http://localhost:8080"
+USERNAME := ""
+PASSWORD := ""
+NUM_CLIENTS := 100
+HATCH_RATE := 20
+
+run:
+	LOCUST_USERNAME=$(USERNAME) LOCUST_PASSWORD=$(PASSWORD) locust --host=$(URL) --no-web --clients=$(NUM_CLIENTS) --hatch-rate=$(HATCH_RATE)
```

Comment:
As I commented in the content-dev meeting, I don't think launching a single locust process with 100 clients will be a realistic scenario. The Python GIL won't allow it to work with the needed speed. 
You can see at https://github.com/learningequality/velox/blob/master/src/lib/locust/wrapper.py#L115-L120 how we did it in velox, to ensure there was one locust slave per client launching requets at the required date. I think you can do a manual test using the cli described at https://docs.locust.io/en/stable/running-locust-without-web-ui.html to check how different the results are. It's not needed to setup different machines, they all can be launched in the same as far as your cpu can deal with it.